examples/dswx_s1_sample_runconfig-v3.0.0-er.1.0.yaml

```diff
@@ -0,0 +1,200 @@
+# Sample RunConfig for use with the DSWx-S1 PGE
+# This RunConfig should require minimal changes in order to be used with the
+# OPERA PCM.
+
+RunConfig:
+    # Name for the RunConfig, may be any string
+    Name: OPERA-DSWX-S1-PGE-SAMPLE-CONFIG
+
+    Groups:
+        # PGE-specific RunConfig section
+        # This section is only used by the PGE, however, paths to inputs/outputs
+        # should align with the similar sections of the SAS RunConfig
+        PGE:
+            PGENameGroup:
+                # Name of the PGE for use with this RunConfig, should always be
+                # DSWX_S1_PGE when using with the DSWx-S1 PGE
+                PGEName: DSWX_S1_PGE
+
+            InputFilesGroup:
+                # List of input files/directories
+                # Must be either a top-level directory containing one or more HLS
```

Comment:
This part of the description looks to be leftover from the DSWx-HLS example, since it references HLS files. It should instead say something like "Must be a list of directories, each of which contains a single RTC burst product .tif and .h5 metadata file"